openbb_sdk/extensions/charting/openbb_charting/core/table.html

```diff
@@ -296,12 +296,12 @@
 `), l = -1, u = -1, c = 0, f = []; c !== s.length; ++c) {
           if (s[c].trim() === "BOT") { f[++l] = [], u = 0; continue } if (!(l < 0)) {
             var d = s[c].trim().split(","), h = d[0], m = d[1]; ++c; for (var p = s[c] || ""; (p.match(/["]/g) || []).length & 1 && c < s.length - 1;)p += `
-`+ s[++c]; switch (p = p.trim(), +h) { case -1: if (p === "BOT") { f[++l] = [], u = 0; continue } else if (p !== "EOD") throw new Error("Unrecognized DIF special command " + p); break; case 0: p === "TRUE" ? f[l][u] = !0 : p === "FALSE" ? f[l][u] = !1 : isNaN(Qr(m)) ? isNaN(Na(m).getDate()) ? f[l][u] = m : f[l][u] = Lt(m) : f[l][u] = Qr(m), ++u; break; case 1: p = p.slice(1, p.length - 1), p = p.replace(/""/g, '"'), p && p.match(/^=".*"$/) && (p = p.slice(2, -1)), f[l][u++] = p !== "" ? p : null; break }if (p === "EOD") break
+`+ s[++c]; switch (p = p.trim(), +h) { case -1: if (p === "BOT") { f[++l] = [], u = 0; continue } else if (p !== "Historical") throw new Error("Unrecognized DIF special command " + p); break; case 0: p === "TRUE" ? f[l][u] = !0 : p === "FALSE" ? f[l][u] = !1 : isNaN(Qr(m)) ? isNaN(Na(m).getDate()) ? f[l][u] = m : f[l][u] = Lt(m) : f[l][u] = Qr(m), ++u; break; case 1: p = p.slice(1, p.length - 1), p = p.replace(/""/g, '"'), p && p.match(/^=".*"$/) && (p = p.slice(2, -1)), f[l][u++] = p !== "" ? p : null; break }if (p === "Historical") break
```

Comment:
Should this change?

Reply:
Nope. I thought the charting was having something custom for EOD but this is End of Data so reverting this too.